cmd/plugin/README.md

```diff
@@ -0,0 +1,23 @@
+# apisix-ingress-controller plugin
+
+The apisix-ingress-controoler plugin can quikly show the apisix routes and upstreams in kubernetes cluster.
+
+# Quikly start 
+
+```shell
+git clone http://github.com/apache/apisix-ingress-controller 
+cd cmd/plugin
+sh install-plugin.sh
```

Comment:
Why not assign the executable permission?

Reply:
oh, right. I add the execcutable permission.

Reply:
beacuse . I used sh executable the install-plugin.sh file. so I don't need executable permission.